### DISCUSSION TITLE
Make develop branch default?

### DISCUSSION THREAD

**Discussion by PiN73** [simolus3/drift#1283] [Ideas]
What do you think about making "develop" branch the default?

This has the following benefits:
- Users who open the repo will see fresh commits and changes by default (which I think is a common use-case when opening github instead of pub.dev or docs)
- Anyone who wants to contribute will make changes on top of the develop branch. This will help to avoid duplication of a feature or a fix which is already made by you, for example
- Pull requests will be created into develop branch by default

Or, alternatively, you can rename master to stable and develop to master

**Comment by simolus3** [comment]
Good call, I've changed the default branch to `develop`.
